cocos/core/splash-screen-webgl.ts

```diff
@@ -385,13 +402,23 @@ export class SplashScreenWebgl {
 
         gl.bindFramebuffer(gl.FRAMEBUFFER, null);
 
-        // TODO: hack for cocosPlay & XIAOMI cause on landscape canvas value is wrong
+        // adapt for cocos play 1.0.11
+        if (COCOSPLAY) {
```

Comment:
Again ? Why keep set innerWidth and innerHeight ?

Reply:
To prevent someone silently modifying

Reply:
In which case exactly ? I don't understand the scenario where we need to check and adapt window size every frame.

Reply:
@LinYunMo and I fixed the problem on 1.0.3, https://github.com/cocos-creator/engine/pull/6107

Not sure if it's needed now
I will test the check is needed or not

Reply:
It is unnecessary on 1.1.1